textproc/dos2unix/Portfile

```diff
@@ -27,9 +26,9 @@ homepage            http://www.xs4all.nl/~waterlan/dos2unix.html
 master_sites        sourceforge:project/dos2unix/dos2unix/${version} \
                     http://www.xs4all.nl/~waterlan/dos2unix/
 
-checksums           rmd160  5eea26d6dbb18310a0e93900a5f10647f0ba88dc \
-                    sha256  b68db41956daf933828423aa30510e00c12d29ef5916e715e8d4e694fe66ca72 \
-                    size    788278
+checksums           rmd160  7952d1dcd768109fd41b4fe88ee99e58677443ba \
+                    sha256  da07788bb2e029b0d63f6471d166f68528acd8da2cf14823a188e8a9d5c1fc15 \
+                    size    959228
 
 depends_lib         port:gettext
 
```

Comment:
the port would likely also benefit from using the "make" PortGroup instead of doing all of it manually